cmd/auth/auth_test.go

```diff
@@ -20,9 +21,13 @@ func TestAuthLoginCmd_Execute(t *testing.T) {
 	rootCmd.SetArgs([]string{"auth", "login"})
 
 	// Execute the command
-	err := rootCmd.Execute()
-	if err != nil {
-		t.Fatalf("Err: %q, Captured StdOut: %q", err, stdout.String())
+	executeErr := rootCmd.Execute()
+	if executeErr != nil {
+		logContent, err := os.ReadFile(os.Getenv("PINGCTL_LOG_PATH"))
```

Comment:
Could we move this to a utility method since it is used across several tests?

Reply:
Moved!